private/templates/default/config.py

```diff
@@ -693,6 +693,8 @@
 #settings.req.requester_optional = True
 # Uncomment if the User Account logging the Request is NOT normally the Requester
 #settings.req.requester_is_author = False
+# Uncommet to enable task field in the list_fields of req_req_skills
```

Comment:
Typo: Uncommet -> Uncomment. Personally I'd prefer a less technical/"code-based" explanation of what this does. Your explanation might confuse users who don't know about the meaning of list_fields.